### PR TITLE
250 Give proper attribution

### PR DESCRIPTION
**Pull request by maxammann**
image

### CODE REVIEW

File: frontend/lib/map/map/attribution_dialog.dart [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      title: Text('Kartendaten'),
+      children: [
+        AttributionDialogItem(
+          icon: Icons.copyright,
+          color: Colors.blue,
+          text: 'OpenStreetMap',
+          onPressed: () {
+            launch("https://www.openstreetmap.org/about/");
+          },
+        ),
```
Comment (elKei24) [inline frontend/lib/map/map/attribution_dialog.dart:22]: According to https://www.openstreetmap.org/copyright, the text should say "© OpenStreetMap-Mitwirkende", and the link should go to https://www.openstreetmap.org/copyright

**Review by elKei24 — APPROVED** [review]

File: frontend/lib/map/map/map.dart [hunk 2]
```diff
@@ -49,28 +51,61 @@ class _MapState extends State<Map> implements MapController {
...
     final config = Configuration.of(context);
-    var statusBarHeight = MediaQuery.of(context).padding.top;
-    var pixelRatio = MediaQuery.of(context).devicePixelRatio;
+    var statusBarHeight = MediaQuery
+        .of(context)
+        .padding
+        .top;
+    var pixelRatio = MediaQuery
+        .of(context)
+        .devicePixelRatio;
```
Comment (Schedulaar) [inline frontend/lib/map/map/map.dart:60]: Can you undo this?

File: frontend/lib/map/map/map.dart [hunk 3]
```diff
@@ -49,28 +51,61 @@ class _MapState extends State<Map> implements MapController {
...
+          children: [
+            MapboxMap(
+              initialCameraPosition: CameraPosition(
+                  target: Map.initialLocation,
+                  zoom: widget.locationAvailable
+                      ? Map.userLocationZoomLevel
+                      : Map.initialZoomLevel),
+              styleString: config.mapStyleUrl,
+              // We provide our own attribution menu
+              attributionButtonMargins: Point(-100, -100),
```
Comment (Schedulaar) [inline frontend/lib/map/map/map.dart:73]: Can you remove the additional iOS margin in map_page.dart?

File: frontend/lib/map/map/map.dart [hunk 4]
```diff
@@ -49,28 +51,61 @@ class _MapState extends State<Map> implements MapController {
...
+              // The Mapbox wordmark must be shown
```
Comment (Schedulaar) [inline frontend/lib/map/map/map.dart:74]:
```suggestion
              // The Mapbox wordmark must be shown because of legal issues
```

**Review by Schedulaar — APPROVED** [review]

**Comment by maxammann** [comment]
@RoAnBu can you test this on ios? Sorry :dagger:

**Pull request merged**